test/integration/crud/misc_cursors.test.js

```diff
@@ -3589,7 +3584,7 @@ describe('Cursor', function () {
     await client.close();
   });
 
-  it('should return implicit session to pool when client-side cursor exhausts results after a getMore', function (done) {
+  it('should return implicit session to pool when client-side cursor exhausts results after a getMore', async function () {
```

Comment:
refactored this test while debugging it but it's not necessary.  we may as well keep it, since it's one more async-await test.